src/components/projects/CreateProjectForm.vue

```diff
@@ -820,8 +820,8 @@ export default {
       max_fits_header_length: 68,
       generic_filter_list: ['Lum', 'Red', 'Green', 'Blue', 'HA', 'O3', 'S2', 'EXO'],
       generic_camera_areas: [
-        '600%', '500%', '400%', '300%', '220%', '133%',
```

Comment:
Did you mean for this change to be in this PR?

Reply:
I thought I reverted the changes. Let me fix!

Reply:
fixed back to what it was!